sources/core/Stride.Core.DiagnosticsServices/ErrorCodes.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace StrideDiagnostics;
+public static class ErrorCodes
+{
```

Comment:
Would be good to add XML comments to each of those constant with some explanation when it is raised.

Reply:
[b2c044c](https://github.com/stride3d/stride/pull/1778/commits/b2c044c34d1961474eaad75ce37169fe8b8e0875)

ive added base comments and renamed them, the documentation is overall really bad for now :D